VideoLocker/res/layout/registration_spinner_item.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    style="?android:attr/spinnerItemStyle"
+    tools:targetApi="17"
+    android:singleLine="true"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:ellipsize="marquee"
+    android:textAlignment="inherit"
+    android:fontFamily="sans-serif"
+    android:paddingTop="16dp"
+    android:textSize="17sp" />
```

Comment:
`@dimen/large` or `@dimen/base` can be used instead of this hardcoded textSize.
p.s. Git loves newlines at end of file.